build/yaml/samples/nsx_v1alpha1_subnet.yaml

```diff
@@ -3,5 +3,5 @@ kind: Subnet
 metadata:
   name: subnet-sample
 spec:
-  accessMode: private
+  accessMode: Private
```

Comment:
could you also add an example for DHCP server subnet?

Reply:
Added, thanks